### DISCUSSION TITLE
Copilot takes up to 20 gigabytes of memory on Linux.

### DISCUSSION THREAD

**Discussion by 88888888oleg** [community/community#110739] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

![Screenshot from 2024-03-02 18-22-53](https://github.com/community/community/assets/46030861/3bf5c5bf-6b2d-4900-b324-77f27100aff5)

**Comment by ayim** [comment]
Which IDE are you using Copilot with?
